sklearn/tree/_tree.pyx

```diff
@@ -622,6 +624,8 @@ cdef class Tree:
 
     def __setstate__(self, d):
         """Setstate re-implementation, for unpickling."""
+        self.capacity = d["capacity"]
```

Comment:
This is overridden below (line 648 in this version of the file). Shall we really have it in the state?